Source/WebCore/rendering/style/StyleCachedImage.cpp

```diff
@@ -146,7 +155,7 @@ RenderSVGResourceContainer* StyleCachedImage::renderSVGResource(const RenderElem
     if (!renderer)
         return nullptr;
 
-    if (imageURL().string().find('#') == notFound)
+    if (m_url.resolved.string().find('#') == notFound)
```

Comment:
```suggestion
    if (!m_url.resolved.string().contains('#'))
```
or if we aren’t worried changing behavior if the URL is invalid
```suggestion
    if (!m_url.resolved.hasFragmentIdentifier())
```